alloptions-limit.php

```diff
@@ -106,87 +122,68 @@ function wpcom_vip_sanity_check_alloptions_notify( $size, $blocked = false ) {
 		return;
 	}
 
-	$irc_alert_level = 2; // ALERT
-
-	$opsgenie_alert_level = 'P4';
-	if ( $blocked ) {
-		$msg = 'This site is now BLOCKED from loading until option sizes are under control.';
-
-		// If site is blocked, then the IRC alert is CRITICAL
-		$irc_alert_level      = 3;
-		$opsgenie_alert_level = 'P3';
-	} else {
-		$msg = 'Site will be blocked from loading if option sizes get too much bigger.';
-	}
-
-	$msg .= "\n\nDebug information can be found in the Fieldguide";
+	/**
+	 * Fires under alloptions warning conditions
+	 *
+	 * @param bool $really_blocked False if alloptions size is large. True if site loading is being blocked.
+	 */
+	do_action( 'vip_alloptions_notification', $really_blocked );
```

Comment:
If I saw `vip_alloptions_notification ` by itself I wouldn't know for sure what it does. Maybe `vip_alloptions_threshold_alert` could be good.